### DISCUSSION TITLE
Hangi kutular boyanmalıdır?

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#4] [Zeka Sorusu] [answered]
![s002](https://github.com/sonsuzus/sonsuzus/assets/52765796/e091c8cc-3804-47ad-9662-7a11c60c47e9)

Bağıntıya göre son bölümde hangi kareler siyaha boyanmalıdır?

**Comment by ArvenAA** [comment, marked as answer]
Eğer boyanan kareler yerine boyanmamışlara bakarsak hepsinin bir sayı oluşturduğunu fark ederiz. 0, 2, 4 ve son olarak 6. Altı olması için ise boyamamız gereken kareler C şıkkında verilmiştir. Yani cevap C şıkkı, 5, 6, 11.
